content/Watsonx/CodeAssistant/101.md

```diff
@@ -0,0 +1,78 @@
+---
+title: '101: Ansible Lightspeed Environment Setup'
+timeToComplete: 15
+updated: 2023-09-13
+---
+
+# 101: Ansible Lightspeed environment setup in VS Code
+
+## Goal
+
+The goal of this lab is to install the Ansible Lightspeed extension and enable and authenticate the extension in VS Code. 
+
+## Install Ansible extension in VS Code
+
+1. Open the VS Code application.
+2. Click the **Extensions** icon on the left sidebar.
+
+    ![](./images/settings-icon.png)
+3. Search for **Ansible** in the search box and then click the **Install** button.
+
+    ![](./images/search-for-ansible.png)
+
+## Enable the Ansible Lightspeed extension
+
+1. After installing the Ansible extension, click on the little gear icon and select the **Extension Setting** option.
+
+    ![](./images/gear-icon.png)
+2. In the Search settings bar, type **lightspeed**
+3. If they are not already selected, check all of the following boxes:
+ - Ansible > Lightspeed: **Enabled**
+ - Ansible > Lightspeed > Suggestions: **Enabled**.
+
+    ![](./images/enable-lightspeed.png)
+
+**Note:** For the purpose of this demo, we will disable Ansible Lint and Ansible Execution Environment.
```

Comment:
I think a common question here would be "Why do we need to disable these items?".  Do we have information around why the code assistant doesn't play nicely with the linter and execution environment?

Reply:
I didn't think Ansible Lint was necessary to showcase the Ansible lightspeed features and would only add more time to complete the lab.

The Ansible Execution Environment requires a container engine, so it would require installing Docker or Podman as a prerequisite. This would add more time to the lab.

<img width="605" alt="Screenshot 2023-09-18 at 1 36 30 PM" src="https://github.com/ibm-build-lab/VAD-VAR-Workshop/assets/19367627/6cd500ee-15c1-4038-bb5e-c27d7981ff4c">
<img width="500" alt="Screenshot 2023-09-18 at 1 35 17 PM" src="https://github.com/ibm-build-lab/VAD-VAR-Workshop/assets/19367627/fcdabce0-8a9a-46ca-aab3-2dea21469080">